packages/devtools_app/lib/src/shared/server_api_client.dart

```diff
@@ -25,25 +26,29 @@ class DevToolsServerConnection {
     initFrameworkController();
   }
 
+  /// Returns a URI for the backend ./api folder for a DevTools page being hosted
```

Comment:
can we add some information to this dart doc regarding where DevTools is served with "devtools" as part of the URI? (i.e. for the standalone VM or Dart CLI app) - see Ben's comment below

Reply:
Done, thanks!

I'll land this when bots finish, but I can tweak the text in another change if desired. Thanks!